### PR TITLE
notes inserted under videos

### LINKED ISSUE #220: Add notes under videos

### PR DESCRIPTION
**Pull request by PhiSpel**
# Description

reopened from #221 
Note sentence added under each video based on page 128 of
https://www.was-ist-oer.de/wp-content/uploads/sites/17/2018/01/Joeran-Muuss-Merholz-Freie-Unterrichtsmaterialien-Beltz-2018.pdf

Videos are in

- [x] all chapters under [Matlab Grundlagenkurs und Zusatzübungen](https://joergbrech.github.io/Modellbildung-und-Simulation/13_matlab/05_matlab_videos/00_matlab_videos.html#grundlagenkurs-und-zusatzubungen)
- [x] [Raketenstart](https://joergbrech.github.io/Modellbildung-und-Simulation/07_num_ode/01_simulation_eines_raketenstarts.html)
- [x] [Vuvuzela Filter](https://joergbrech.github.io/Modellbildung-und-Simulation/10_dft/04_vuvuzela_filter.html)
- [x] [Doppler Effekt](https://joergbrech.github.io/Modellbildung-und-Simulation/10_dft/03_der_doppler_effekt.html)
- [x] [Highline](https://joergbrech.github.io/Modellbildung-und-Simulation/05_lineare_gleichungen/01_modellierung_einer_highline.html)

closes #220 

## Checklist

 - [x] This pull request is associated to an issue
 - [x] All used resources use a CC-by-SA license or less and are marked appropriately, see [our contribution guide](CONTRIBUTING.md).
 - [x] All used resources have been added to the list in `[TODO, does not exist yet]`
 - [x] New exercises include learning requirements and learning goals.

### CODE REVIEW

File: content/04_lineare_gleichungen/01_modellierung_einer_highline.ipynb [hunk 1]
```diff
@@ -9,6 +9,8 @@
     "\n",
     "<iframe width=\"560\" height=\"315\" src=\"https://www.youtube.com/embed/C6MtzvQ5hZ8\" frameborder=\"0\" allow=\"accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture\" allowfullscreen></iframe>\n",
     "\n",
+    "<div style=\"text-align: left\";\"color: grey\"> - Hinweis: Dieses Video ist von YouTube aus eingebunden und nicht Teil des frei lizenzierten Materials! - </div> <br>\n",
```
Comment (m-r-schenk) [inline content/04_lineare_gleichungen/01_modellierung_einer_highline.ipynb:12]: Why color: grey? This is the only Video caption in grey I think.

**Review by m-r-schenk — APPROVED** [review]
Ok for me, but we should have consistent captions. Not have one in grey and the rest in black.

File: content/04_lineare_gleichungen/01_modellierung_einer_highline.ipynb (hunk 1 above)
Comment (PhiSpel) [reply content/04_lineare_gleichungen/01_modellierung_einer_highline.ipynb]: As the captions are not part of the exercise but a necessity, I would keep them rather small. I only changed one file so you could see what it looks like and we can agree before changing the other captions. What do you think about [579bb09](https://github.com/joergbrech/Modellbildung-und-Simulation/pull/247/commits/579bb096675825a9710e508d8f4a9afde149a4f6)? Like this, it is much less intrusive. If you like it, I can quickly change the other captions and we can merge

**Review by m-r-schenk — APPROVED** [review]
OK

**Pull request merged**